build.gradle.kts

```diff
@@ -20,4 +20,5 @@ plugins {
     id("com.android.library") version "7.3.0" apply false
     id("org.jetbrains.kotlin.android") version "1.7.10" apply false
     id("org.jetbrains.kotlin.jvm") version "1.7.10" apply false
+    id("androidx.navigation.safeargs.kotlin") version "2.5.3" apply false
```

Comment:
I personally see no usage of the navigation component and safe args here, as I mentioned before we don't need this as a running application but rather code samples only as we're building a **static** analysis tool

Reply:
As previously discussed in a call, I initially thought of building a small complete sample app as a demo showcasing the issues that can occur when one violates the lint checks we have, but given this may make it harder to maintain whenever we add new lint checks, we agreed to move forward with just highlighting the lint issues in separate files.